src/components/ButtonsComponent.jsx

```diff
@@ -19,38 +19,40 @@ class ButtonsComponent extends React.Component {
    	});
   }
 
-  handleChangeImportance(value){
+  handleChangeImportance(e){
     this.setState({
-   	  importance: value
+   	  importance: e.target.value
     });
   }
 
-  handleSubmit(text, importance){
+  handleSubmit(text, importance, Click){
     if(!this.state.text.length){
       return;
     }
-    this.props.submit(text, importance);
+    Click(text, importance);
```

Comment:
what is the reason of passing Click handler as argument? I believe you can just use this,props.Click